### PR TITLE
Adding Standard_Microsoft in Cdn Pricing sku

### PR DESCRIPTION
**Pull request by hytao**
## Description

Adding Standard_Microsoft in CDN pricing sku.

Old PR link https://github.com/Azure/azure-powershell/pull/7200
Swagger PR link https://github.com/Azure/azure-rest-api-specs/pull/3628
SDK PR link https://github.com/Azure/azure-sdk-for-net/pull/4730
## Checklist

- [x ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x ] The title of the PR is clear and informative
- [ x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/Cdn/Commands.Cdn/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,9 @@
 -->
 ## Current Release
 
+## Version 5.0.6
```
Comment (praries880) [inline src/ResourceManager/Cdn/Commands.Cdn/ChangeLog.md]: remove the version header, leave the change description directly under `Current Release`

**Comment by praries880** [comment]
ping @hytao

**Comment by maddieclayton** [comment]
@hytao Code complete is today, please update - we will not accept changes after today for Ignite.

File: src/ResourceManager/Cdn/Commands.Cdn/ChangeLog.md (hunk 1 above)
Comment (hytao) [reply src/ResourceManager/Cdn/Commands.Cdn/ChangeLog.md]: I'll fix the test. Thanks!

**Pull request merged**